guides/administration/using-nsenter-to-access-a-node-without-ssh.md

```diff
@@ -0,0 +1,41 @@
+# Using nsenter to access a node without SSH
+
+In addition SSH, [nsenter](https://github.com/jpetazzo/nsenter) can be used to gain root access to a node. This method does NOT require any password or authentication, so be sure that your Kontena grid users are trusted.
+
+First you need to install the `nsenter` as a daemon service so that it runs on each node. There is a sample stack in the Kontena Stack Registry:
+
```

Comment:
Needs formatting with backtics

Reply:
It renders correctly with double indents, is there some other reason to use backticks?

![using_nsenter_to_access_a_node_without_ssh_ _kontena](https://user-images.githubusercontent.com/3893/35852291-2c74e538-0b33-11e8-9504-58fdfef6c299.png)

Reply:
No other reason than my ignorance. :D Learned something new. :)